### DISCUSSION TITLE
I need to add another step to bitbucket pipelines yml

### DISCUSSION THREAD

**Discussion by ramesh82** [mikefarah/yq#1352] [Q&A]
How could I add another step to existing bitbucket pipelines yml file?

This is the current file
``default:
  - parallel:
    - step:
        name: build something
        script:
          - command1
          - command2

And I want to add another step to it. Step can be copied from another file but need to replace the environment variable.
``default:
  - parallel:
    - step:
        name: to be added
        script:
          - new command ${env:variable}

Result should be something like this
``default:
  - parallel:
    - step:
        name: build something
        script:
          - command1
          - command2
    - step:
        name: to be added
        script:
          - new command run

**Comment by mikefarah** [comment]
It's very hard to read your yaml examples, can you please fix the formatting see the github guides for help.
